packages/@react-spectrum-uxp/parcel-resolver-alias-fix/AliasFixResolver.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const { Resolver } = require('@parcel/plugin');
+const NodeResolver = require('@parcel/node-resolver-core').default;
+const path = require('path');
+
+const { Console } = require('console');
+const console = new Console(process.stdout, process.stdin);
+
+async function getPackage(fs, file) {
+  let parent = path.dirname(file);
+  if (parent === file) {
+    return [];
+  }
+  let check = path.join(parent, "package.json");
+  if (fs.existsSync(check)) {
+    return [parent, JSON.parse((await fs.readFile(check)).toString())];
+  }
+  return getPackage(fs, parent);
+}
+
+//  this resolver just finds the "alias" entries in the closest package.json from the entryRoot.
```

Comment:
There's an issue for upgrading parcel, do you think you could add a link into that issue or tag yourself in it so that you're updated when we do that?

Reply:
https://github.com/adobe/react-spectrum/issues/1743

Reply:
Tagged myself.